.github/workflows/all-in-one-build.yml

```diff
@@ -0,0 +1,35 @@
+name: Docker
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  all-in-one:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+
+    - uses: actions/setup-go@v2
+      with:
+        go-version: ^1.15
+
+    - uses: docker/login-action@v1
+      with:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+    - uses: nelonoel/branch-name@v1.0.1
```

Comment:
I don't like to depend on an external action, and a pretty complicated at that, for something that we do with a single shell line https://github.com/jaegertracing/jaeger/blob/556e3ba94bcf9742ceb6ffc53f806647de3b0e64/.travis.yml#L67

I recommend defining our own small GH action that we can reuse in multiple workflows, as I did in https://github.com/jaegertracing/jaeger-client-go/pull/552. That example also shows how to export an env variable by writing into a special GH file.

Reply:
Sure this sounds good.